flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/source/TestStreamingMonitorFunction.java

```diff
@@ -113,7 +114,7 @@ public void testConsumeWithoutStartSnapshotId() throws Exception {
 
       Assert.assertTrue(
           "Should have expected elements.", latch.await(WAIT_TIME_MILLIS, TimeUnit.MILLISECONDS));
-      Thread.sleep(1000L);
+      Awaitility.await().atLeast(1, TimeUnit.SECONDS).until(() -> true);
```

Comment:
this goes back to my earlier comment [here](https://github.com/apache/iceberg/pull/8715#issuecomment-1748190639). We don't want to blindly just change Thread.sleep usage to Awaitility

Reply:
will it be ok if i replace this with simply Awaitility.await()
        .atMost(WAIT_TIME_MILLIS, TimeUnit.MILLISECONDS)
        .pollInterval(100, TimeUnit.MILLISECONDS)
        .until(() -> latch.getCount() == 0);